buildcomponents.yaml

```diff
@@ -0,0 +1,12 @@
+.migrate:
+  script:
+    - cp -r $CI_PROJECT_DIR/sql/* /flyway/sql/
```

Comment:
I had a change to ensure the /flyway/sql directory existed... not positive that's necessary, but making sure you considered it.